website/content/commands/license.mdx

```diff
@@ -56,6 +59,10 @@ Subcommands:
 
 ## put
 
+-> **Deprecated** The ability to manage the cluster's license via the CLI
```

Comment:
Is the plan to remove these deprecated subcommands in a later release?

Reply:
Almost certainly yes. For now they are left in place but will always throw errors indicating the removal of api/cli driven licensing. I could add a blurb about them being fully removed in a future release.

Reply:
That would be useful, to set expectations